README.md

```diff
@@ -1,13 +1,13 @@
-# Collabo Web App
+# collabo-community-app
 
 [![contributions welcome](https://img.shields.io/badge/contributions-welcome-brightgreen.svg?style=flat)](https://code-collabo.gitbook.io/collabo-contributor/collabo-web-app-project/contribute-to-collabo-web-app) [![GitHub pull requests](https://img.shields.io/github/issues-pr/code-collabo/collabo-web-app?color=goldenrod)](https://github.com/code-collabo/collabo-web-app/pulls)
 
-The Collabo Web App is an internal project aimed at promoting the Code Collabo Community, its projects, and programs. It also serves as a showcase for our web development tools like node-mongo and Collabo CSS framework. Opportunities for UI/UX and graphic design contributions are also available.
+The Collabo community App is an internal project aimed at promoting the Code Collabo Community, its projects, and programs. It also serves as a showcase for our web development tools like node-mongo and Collabo CSS framework. Opportunities for UI/UX and graphic design contributions are also available.
```

Comment:
**_Review comment 2:_** "The Collabo community App" -  Make the word "community" begin with a capital letter similar to what was there before @Dkingofcode

Reply:
Will endeavour to make those changes appropriately.